pkg/drivers/fakedriver/fakedriver.go

```diff
@@ -0,0 +1,72 @@
+package fakedriver
```

Comment:
Used by `pkg/libmachine/host/host_test.go`

Reply:
Removed. It was not bringing much info.